.travis.yml

```diff
@@ -37,7 +37,7 @@ matrix:
       before_script:
         - eslint --ext .json .
         - jshint .
-        - validate -i common-schema.json -i geometry-schema.json -i specs/Weather/weather-schema.json -i specs/Alert/alert-schema.json -p . -c true
+        - node ./validator/validate.js -i common-schema.json -i geometry-schema.json -i specs/Weather/weather-schema.json -i specs/Alert/alert-schema.json -p . -c true
```

Comment:
```validate -i common-schema.json -i geometry-schema.json -i specs/Weather/weather-schema.json -i specs/Alert/alert-schema.json -p . -c true```

using install -g you can keep it unchanged

Reply:
see above